api/src/main/java/org/openmrs/module/kenyaemr/reporting/data/converter/definition/cacx/CACXPopulationTypeDataDefinition.java

```diff
@@ -0,0 +1,47 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.module.kenyaemr.reporting.data.converter.definition.cacx;
+
+import org.openmrs.module.reporting.data.BaseDataDefinition;
+import org.openmrs.module.reporting.data.person.definition.PersonDataDefinition;
+import org.openmrs.module.reporting.definition.configuration.ConfigurationPropertyCachingStrategy;
+import org.openmrs.module.reporting.evaluation.caching.Caching;
+
+/**
+ * CACX Population Type Column
+ */
+@Caching(strategy=ConfigurationPropertyCachingStrategy.class)
+public class CACXPopulationTypeDataDefinition extends BaseDataDefinition implements PersonDataDefinition {
```

Comment:
Am glad this implements PersonDataDefinition and so it can be used across other reports. 
Can we instead used a generic name i.e. LastPopulationType or something so that it is used across?